Remove empty horizon component from `irradiance.haydavies` (#2788)
pvlib/pvlib-python: docs/sphinx/source/whatsnew/v0.16.0.rst

.. _whatsnew_0_16_0:

v0.16.0
-----------------------

Breaking Changes
~~~~~~~~~~~~~~~~

> * Remove empty ``poa_horizon`` component from the output of

Reviewer: ```suggestion
* Remove empty ``poa_horizon`` key from the ``diffuse_components`` output of
```

>   :py:func:`pvlib.irradiance.haydavies` when ``return_components=True``.

Reviewer: ```suggestion
  :py:func:`pvlib.irradiance.haydavies`.
```

Reviewer: These are grammar clarifications.

> (:pull:`2788`)

Reviewer 2: ```suggestion
* Remove empty ``poa_horizon`` component from the output of
  :py:func:`pvlib.irradiance.haydavies` when ``return_components=True``.
  (:pull:`2788`)
```
